README.md

```diff
@@ -30,98 +12,45 @@ Command for running postfix docker container:
 ```
 docker run -it -e MYHOSTNAME=localhost \
     -p 25:10025\
-    -v $(pwd)/tmp/postfix:/var/spool/postfix \
-    -v $(pwd)/tmp/mail:/var/spool/mail postfix
+    -v /var/spool/postfix:/var/spool/postfix \
+    -v /var/spool/mail:/var/spool/mail postfix
+    -d docker.io/modularitycontainers/postfix
 ```
 
 ## How to use the container over standard TLS
 
-Dockerfile for TLS port is Dockerfile.TLS.
-
 Command for running postfix docker container:
 ```
-docker run -it -e DEBUG_MODE=yes -e MYHOSTNAME=localhost \
+docker run -it -e ENABLE_TLS -e DEBUG_MODE=yes -e MYHOSTNAME=localhost \
     -p 25:10025 \
-    postfix
+    postfix -d docker.io/modularitycontainers/postfix
 ```
 
-Environment variable DEBUG_MODE is used for debugging proposes
-from Postfix and dovecot point of views.
+Environment variable DEBUG_MODE is used for debugging purposes
+from postfix point of view.
 
-## How to use the container with IMAP with TLS enabled (dovecot and Postfix)
-
-Dockerfile which uses IMAP is Dockerfile.IMAP.
+## How to use the container with IMAP.
 
 Command for running postfix docker container:
 ```
-docker run -it -e MYHOSTNAME=localhost -e DEBUG_MODE \
-    -p 25:10025 -p 587:10587 -p 143:10143 \
-    postfix
+POSTFIX_CERTS_PATH=/etc/postfix/certs
+docker run -it -e ENABLE_IMAP -e MYHOSTNAME=localhost
+    -e DEBUG_MODE \
```

Comment:
I'm confused: it's `-e DEBUG_MODE` here and `-e DEBUG_MODE=yes` above, can we have this unified?